docs/setup.md

```diff
@@ -169,7 +169,7 @@ Add the following imports:
 #import "AppFramework/Action/GREYAction.h"
 #import "AppFramework/Action/GREYActionBlock.h"
 #import "AppFramework/Action/GREYActions.h"
-#import "AppFramework/Matcher/GREYElementMatcherBlock.h"
+#import "CommonLib/Matcher/GREYElementMatcherBlock.h"
```

Comment:
Maybe send a separate PR for this?

Reply:
Done: https://github.com/google/EarlGrey/pull/791